internal/cmd/commands/scopescmd/funcs.go

```diff
@@ -9,6 +9,12 @@ import (
 	"github.com/hashicorp/boundary/internal/cmd/base"
 )
 
+const (
```

Comment:
Good thought. I should have done this when refactoring but just didn't think about it. We could update that over time though.